Src/Particle/AMReX_NeighborParticlesI.H

```diff
@@ -141,7 +141,7 @@ NeighborParticleContainer<NStructReal, NStructInt, NArrayReal, NArrayInt>
         {
             const Geometry& geom = this->Geom(lev);
 
-            mask_ptr[lev].reset(new iMultiFab(ba, dmap, num_mask_comps, m_num_neighbor_cells));
+            mask_ptr[lev] = std::make_unique<iMultiFab>(ba, dmap, int(num_mask_comps), m_num_neighbor_cells);
```

Comment:
For reasons I don't completely understand, we have to use `int(num_mask_comps)` here for debug build.  The reason might be this function is not inline in debug mode and `num_mask_comps` is a `static constexpr` member that has no definition.  Thus it cannot be odr-used before C++17.  `std::make_unique` is probably trying to match `num_mask_comps` to `const int&` (an odr-use).